duckduckgo/iOS
# show unencrypted warning message

Task/Issue URL:  https://app.asana.com/0/414235014887631/312386120368414
Tech Design URL:
CC:

**Description**:

Shows enhanced messaging for unencrypted connections. 

See Zeplin:  https://app.zeplin.io/project/59d519a58458d74fd8a6d320/screen/5a4eb64a79130c24d4bd13e3

**Steps to test this PR**:

1.  Visit thehill.com
1. Open Privacy Dashboard, drill down in to Encryption detail
1. Observe message

1. Visit duckduckgo
1. Open Privacy Dashboard, drill down in to Encryption detail
1. Confirm encrypted screen still looks good


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
